docs/alkiln/writing_tests.mdx

```diff
@@ -2477,7 +2477,7 @@ If your package is leaving out the al_package.yml file from the Assembly Line pa
 
 ### Testing unfinished code {#unstable-tip}
 
-If you are still developing your interview, you can still write a [simple test](setup.mdx/#first-test) to make sure your interview at least loads without an error. You might want to wait till your code is more stable to write more tests.
+If you are still developing your interview, you can still write a [simple test](setup#.mdx#first-test) to make sure your interview at least loads without an error. You might want to wait till your code is more stable to write more tests.
```

Comment:
actually the problem is the .MDX short cross reference style, it's no longer supported. You should delete the .MDX and second #

Reply:
I will fix this one when fixing the admonitions. Also, see #414 which I just opened to make sure these are all consistent. My focus for now was to get working links so that the site would build. I'm expecting a small number of regressions and I'm OK with that vs exhaustive testing.